R/weight.R

```diff
@@ -59,3 +59,103 @@ setMethod("weightVariables", "VariableCatalog", function (x) {
         return(c())
     }
 })
+
+#' Generate a weight vector
```

Comment:
"variable", not "vector". This is creating an expression for a derived variable.